scripts/build_html.py

```
@@ -79,15 +85,23 @@ def sanitize_identifier(name: str) -> str:
 
                 # Write bytes in rows of 16
                 for i in range(0, len(compressed), 16):
-                    chunk = compressed[i:i+16]
-                    hex_values = ', '.join(f'0x{b:02x}' for b in chunk)
+                    chunk = compressed[i : i + 16]
+                    hex_values = ", ".join(f"0x{b:02x}" for b in chunk)
                     h.write(f"  {hex_values},\n")
 
                 h.write(f"}};\n\n")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)